README.md

```diff
@@ -1,2 +1,255 @@
+
+Sensor Cloud Eval is a **TSX version** of MODE's Sensor Cloud, developed to be used as an evaluation kit by business partners and app developers. The codebase outlines the latest SPA development design from React / Typescript, including [context](https://reactjs.org/docs/context.html) and [flux-style data flow](https://facebook.github.io/flux/), [hooks](https://reactjs.org/docs/hooks-intro.html), [websockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_client_applications), and more.
+
+## Contents
+
+- [Features](#features)
+- [Installation and Setup](#installation-and-setup)
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+- [Mode API Reference](https://dev.tinkermode.com/docs/api/)
+- [App Navigation](#app-navigation)
+  -[Link and Unlink Sensor Modules to and from a Gateway](#link-and-unlink-sensor-modules-to-and-from-a-gateway)
+  -[Viewing and Manipulating the Sensor Module](#viewing-and-manipulating-the-sensor-module)
+- [API Examples](#api-examples)
+  - [Using REST Endpoints (GET, POST, PUT, PATCH, DELETE)](#using-rest-endpoints)
+    - [Handling User Information](#handling-user-information)
+    - [Linking / Unlinking Sensor Modules](#linking-and-unlinking-sensor-modules)
+    - [Sending Device Commands](#sending-device-commands)
+  - [Receiving Device Events (Websocket)](#receiving-device-events)
+- [Acknowledgements](#acknowledgements)
+- [Links](#links)
+
+## Features
+
+- Management of sensor modules and their device associations from within the application. :electric_plug:
+
+- Ability to view real-time sensor data in multiple sensor modules across multiple devices. :chart_with_upwards_trend:
+
+- Ability to view time-series sensor data and toggle between various time-horizons as well as sensing intervals. :clock10:
+
+- Configuration of sensor module sensing and general settings. :arrows_counterclockwise:
+
+## Installation and Setup
+
+To begin development off MODE's sensor cloud evaluation kit, first clone the repository to your local machine.
+```sh
+$ git clone https://github.com/moderepo/sensor-cloud-eval.git
+```
+
+## Prerequisites
+
+1. You will need MODE, Inc. to pre-provision the gateway(s) to your project before running the application
+
+2. You will also need to have any sensors you wish to pair with your gateway(s) in your possession.
+
+3. You must have an account with the [Tinkermode Console](https://console.tinkermode.com/console/signup).
+
+4. If you don't already have npm and typescript installed, you will need to install both in order to run the application. You will also need at least `Node 8.16.0` or `Node 10.16.0` or later version on your local development machine. If you don't already have node, please navigate [here](https://nodejs.org/en/download/) to download the source code compatible with your machine.
+
+```sh
+$ npm install -g npm
+$ npm install -g typescript
+$ npm install -g node
```

Comment:
I think it's not necessary because `node` is installed by another way you described on the above.
And `npm` is installed with `node`.
In what case is `npm install -g node` used?